web.go

```diff
@@ -28,6 +30,7 @@ type WebProvider struct {
 	KeyFile  string `description:"SSL certificate"`
 	ReadOnly bool   `description:"Enable read only API"`
 	server   *Server
+	Auth     *types.Auth `description:"Auth Crendetials for Web Point"`
```

Comment:
@cocap10 great catch !
Indeed, it sounds OK to assume that TOML will be used to configure authentication users.
Let's remove the struct tag for now :)